tests/doc_comment.rs

```diff
@@ -0,0 +1,22 @@
+extern crate snafu;
+
+use snafu::Snafu;
+
+#[derive(Debug, Snafu)]
+enum Error {
+    /// No user available.
+    /// You may need to specify one.
+    ///
+    /// Here is a more detailed description.
+    MissingUser,
```

Comment:
Can we add one more variant here that shows that `#[snafu(display)]` overrides a doc comment?